### PR TITLE
multitime 1.4 (new formula)

### PR DESCRIPTION
**Pull request by milot-mirdita**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

I tried to get this tool into homebrew-core before in https://github.com/Homebrew/homebrew-core/pull/58568. It hadn't reached any of the notability threshold yet. It now finally managed to do so. I really like this tool and would like to see it be included in the default tap.

### CODE REVIEW

File: Formula/multitime.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}",
+                          "--mandir=#{man}"
+
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/multitime", "-n", "2", "sleep", "1"
```
Comment (carlocab) [inline Formula/multitime.rb:22]: Can we do an `assert_match` or, better, `assert_equal` to check the output here?

Comment (milot-mirdita) [reply Formula/multitime.rb:22]: I don't think that would work very well as the output can very a little bit on the last decimal position.
e.g:
```
===> multitime results
1: sleep 1
            Mean        Std.Dev.    Min         Median      Max
real        1.004       0.001       1.003       1.004       1.006
user        0.001       0.000       0.000       0.001       0.001
sys         0.002       0.001       0.001       0.002       0.003
```

I could check that the first line is present, but that doesn't seem super useful.

Comment (carlocab) [reply Formula/multitime.rb:22]: `assert_match` can take regex arguments.

Comment (milot-mirdita) [reply Formula/multitime.rb]: How about this:
```
output = shell_output("#{bin}/multitime -n 2 sleep 1 2>&1")
assert_match(/((real|user|sys)\s+(\d\.\d\d\d\s*)+){3}/m, output)
```

(I force pushed this change to my branch.)

**Review by dawidd6 — DISMISSED** [review]

File: Formula/multitime.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}",
+                          "--mandir=#{man}"
+
+    system "make", "install"
+  end
+
+  test do
+    output = shell_output("#{bin}/multitime -n 2 sleep 1 2>&1")
+    assert_match(/((real|user|sys)\s+(\d\.\d\d\d\s*)+){3}/m, output)
```
Comment (carlocab) [inline Formula/multitime.rb:23]:
```suggestion
    assert_match(/((real|user|sys)\s+([01]\.\d\d\d\s*)+){3}/m, output)
```
Let's make this a bit more specific. I'm assuming anything other than a `0` or `1` there suggests that something's broken.

Comment (milot-mirdita) [reply Formula/multitime.rb]: Sounds good i further cleaned up/tightened the regex to  `/((real|user|sys)\s+([01].\d{3}\s*){5}){3}/m`

**Review by carlocab — APPROVED** [review]
Nice work. Thanks, @milot-mirdita!

**Pull request closed without merging**